Update gdpr-compliance-doc.md (#48)
usegalaxy-be/usegalaxy-be.github.io: gdpr-compliance-doc.md

And how to exercise them in the context of the Galaxy Service provided by the Galaxy Belgium Team and the Belgium Galaxy Community.

### Right to Access to Personal Information
The only personal information we store that is associated with you, you can see on [https://usegalaxy.be/user/information](https://usegalaxy.be/user/information). We have logs containing IP addresses but these logs are destroyed after 24 hours and are not associated with a single user account.

### Right to Rectification of Personal Information
You can do this on [https://usegalaxy.be/user/information](https://usegalaxy.be/user/information).

### Right to Erasure

> You can do this yourself via the 'Delete Account' option found in your 'User Preferences': [https://usegalaxy.be/user](https://usegalaxy.be/user).

Reviewer: ```suggestion
You can delete your account yourself via the 'Delete Account' option found in your 'User Preferences': [https://usegalaxy.be/user](https://usegalaxy.be/user).
```

### Right to Restrict Processing
All of the processing activities we currently do are required to use the service. We do not transmit any Personally identifiable information (PII) to third parties.

### Right to Data Portability

> The upstream software Galaxy currently does not support this workflow, but does expose an interface that would allow us to do this. The Galaxy Project is [working on the implementation](https://github.com/galaxyproject/galaxy/issues/6082).

Reviewer: ```suggestion
The upstream software Galaxy currently does not support this workflow, but does expose an interface that would allow us to do this. The [Galaxy Project](https://galaxyproject.org/) is working on the implementation.
```
